### DISCUSSION TITLE
Turn off auto-scroll with new entry

### DISCUSSION THREAD

**Discussion by c-fitzmaurice** [spatie/ray#423] [Ideas]
Hey all, LOVE THE APP!!

Just a small request: 

I'm working with queued jobs and need to dump a bunch of text to Ray. It would be great if you could turn off auto-scrolling when a new entry comes in. I have to fight Ray to keep my data that I want to see in the view.

**Comment by freekmurze** [comment]
We'll add this in a future release!

**Reply by c-fitzmaurice** [reply]
Thanks so much for the quick reply, much appreciated!
